.github/workflows/generate_sample.yml

```diff
@@ -0,0 +1,42 @@
+name: Generate sample project
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  test:
+    name: Generate sample project
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout
```

Comment:
(Optional)
```suggestion
      - name: Check out
```

Reply:
Fixed in https://github.com/nimblehq/flutter-templates/pull/196/commits/be22efc9cd04ee3e2bf3bb8d95bc1e13a148e5ef